.github/workflows/workflow.yml

```diff
@@ -105,7 +112,24 @@ jobs:
       env:
         NAVITIA_DOCKER_NETWORK: ${{ job.container.network }}
         TYR_CELERY_BROKER_URL: 'amqp://guest:guest@rabbitmq:5672//'
-
+    - name: SonarCloud Scan
+      if: matrix.docker_image == 'navitia/debian10_dev' # We only want 1 scan as it would bloat sonarcube otherwise
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+      run: |
+          sonar-scanner-4.6.0.2311-linux/bin/sonar-scanner \
```

Comment:
We call `sonar-scanner` manually because of some limitation on c++ with the sonar-scanner-docker being used by the github action - https://github.com/SonarSource/sonarcloud-github-action : 
 > https://github.com/SonarSource/sonarcloud-github-action#do-not-use-this-github-action-if-you-are-in-the-following-situations
> Do not use this GitHub action if [...] You want to analyze C/C++ code

The problem is that [docker-scanner-cli](https://hub.docker.com/r/sonarsource/sonar-scanner-cli) docker image doesn't support well c++ :  
> Warning: These Docker images are not compatible with C/C++/Objective-C projects.